keda/Dockerfile

```diff
@@ -26,4 +26,6 @@ ENV OPERATOR=/usr/local/bin/keda \
 
 ENTRYPOINT ["/usr/local/bin/entrypoint"]
 
+RUN yum -y update gnutls
```

Comment:
@andriisoldatenko This does not look like a NATS change?

Reply:
done https://github.com/astronomer/ap-vendor/pull/40